google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineCloudType.java

```diff
@@ -91,7 +92,7 @@ public AppEngineServerConfiguration createDefaultConfiguration() {
   @Override
   public RemoteServerConfigurable createServerConfigurable(
       @NotNull AppEngineServerConfiguration configuration) {
-    return new AppEngineCloudConfigurable(configuration, null);
```

Comment:
Interesting question. If we proxied it, it would take care of @aslo point about backwards compatibility (with users that previously had a server configuration). The only downside I see here is duplication.